client/src/hooks/useAuth.ts

```diff
@@ -3,9 +3,9 @@ import { useSelector } from "react-redux";
 import { selectUser } from "@Store/reducers/userReducer";
 
 export const useAuth = () => {
-  const { user, isLoading } = useSelector(selectUser);
+  const { user } = useSelector(selectUser);
 
   const userId = user ? user.id : null;
 
-  return { isAuth: !!user, user, userId, isLoading };
+  return { isAuth: !!user, user, userId };
```

Comment:
Does it make sense to return userId separately if you return user object ?

Reply:
remove userId